.buildkite/Dockerfile

```diff
@@ -48,7 +48,7 @@ RUN echo "ulimit -c 0" >> /root/.bashrc
 
 # Setup Bazel caches
 RUN (echo "build --remote_cache=${REMOTE_CACHE_URL}" >> /root/.bazelrc); \
-    (if [ "${BUILDKITE_PULL_REQUEST}" != "false" ]; then (echo "build --remote_upload_local_results=false" >> /root/.bazelrc); fi); \
```

Comment:
we need to keep the original flag as well. This is built so we are not uploading binary asserts to the cache.

Reply:
Thanks for the info! Updated.